tests/e2e/performance/test_vdbench_workload.py

```diff
@@ -57,6 +63,20 @@ def teardown():
     worker_nodes = helpers.get_worker_nodes()
     # Getting list of free nodes
     free_nodes = list(set(worker_nodes) - set(ocs_nodes))
+    if not free_nodes:
+        # Getting list of current machinesets
+        log.info('Adding new worker node for the application to run on.')
+        mset = machine.get_machinesets()
+        replica_count = machine.get_replica_count(mset[0])
```

Comment:
Does `mset[0]` always represent worker node?

Reply:
> > As discussed offline,
> > I don't see a reason to add the 4th node as this is not mimic a customer use case.
> > Our customers will run on OCP cluster containing 6 nodes - 3 masters, 3 workers.
> > OCS node will run on the 3 worker nodes along with the customer applications.
> > VDbench is a customer application in our scenario and therefore should run on 1 of the 3 worker nodes.
> > If I got something wrong here, please reply
> 
> There are 2 ways applications can be run by customers.
> 
> 1. Running applications along with OCS in converged mode
> 2. Running applications in other nodes from OCS nodes. i.e., OCS runs on dedicated nodes
>    (There are other options like running OCS on master nodes, but let's keep it aside for now)
> 
> Option 1 is typically chosen by small customers AFAIU who don't want to have a higher number of nodes. But the catch here is that the resources for `m5.4xlarge` i.e., 64 GB RAM and 32 CPU are to be made available for OCS and depending on the number of applications, more resources need to be made available. So, if we are to run performance tests on just 3 nodes, we should be using a bigger instance type.
> 
> On the other hand, Option 2 will be the widely used model, where OCS runs on dedicated nodes with m5.4xlarge instance type and applications will run on other nodes. This allows customers to scale up app nodes when there is demand and scale down when the demand is less. This will be a widely used model and it is important we run scale and performance tests this way.
> 
> So, running vdbench or FIO on the 4th node is a realistic scenario IMO. Please let me know if this makes sense.

I will add a parameter for selecting if the test need to run on different node, or on the same as the OCS.